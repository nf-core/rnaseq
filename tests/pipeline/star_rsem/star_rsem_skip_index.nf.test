nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"
    options "-profile test,docker"
    tag "pipeline"

    test("STAR RSEM With Salmon (Skip Index) with Salmon (GSE110004)") {

        when {
            params {
                outdir         = "$outputDir"
                pseudo_aligner = "salmon"
                aligner        = "star_rsem"
                rsem_index     = false
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(path("$outputDir/pipeline_info/software_versions.yml")).match("software_versions") },
                { assert snapshot(path("$outputDir/pipeline_info/samplesheet.valid.csv")).match("samplesheet") },
                { assert snapshot(path("$outputDir/bbsplit/RAP1_IAA_30M_REP1.stats.txt"),
                                path("$outputDir/bbsplit/RAP1_UNINDUCED_REP1.stats.txt"),
                                path("$outputDir/bbsplit/RAP1_UNINDUCED_REP2.stats.txt"),
                                path("$outputDir/bbsplit/WT_REP1.stats.txt")).match("bbsplit") },
                { assert new File("$outputDir/bbsplit/WT_REP2.stats.txt").exists() },
                { assert snapshot(path("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_cutadapt.txt"),
                                path("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_fastqc.txt"),
                                path("$outputDir/multiqc/star_rsem/multiqc_data/picard_histogram.txt")).match("multiqc") },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_rsem.txt").exists() },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_picard_dups.txt").exists() },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_general_stats.txt").exists() },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_samtools_flagstat.txt").exists() },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_samtools_idxstats.txt").exists() },
                { assert new File("$outputDir/multiqc/star_rsem/multiqc_data/multiqc_samtools_stats.txt").exists() },
                { assert snapshot(path("$outputDir/salmon/deseq2_qc/R_sessionInfo.log"),
                                path("$outputDir/salmon/salmon_tx2gene.tsv")).match("salmon") },
                { assert new File("$outputDir/salmon/salmon.merged.gene_counts.tsv").exists() },
                { assert new File("$outputDir/salmon/salmon.merged.transcript_counts.tsv").exists() },
                { assert new File("$outputDir/salmon/RAP1_IAA_30M_REP1/quant.sf").exists() },
                { assert new File("$outputDir/salmon/RAP1_IAA_30M_REP1/quant.genes.sf").exists() },
                { assert new File("$outputDir/salmon/RAP1_UNINDUCED_REP1/quant.sf").exists() },
                { assert new File("$outputDir/salmon/RAP1_UNINDUCED_REP1/quant.genes.sf").exists() },
                { assert new File("$outputDir/salmon/RAP1_UNINDUCED_REP2/quant.sf").exists() },
                { assert new File("$outputDir/salmon/RAP1_UNINDUCED_REP2/quant.genes.sf").exists() },
                { assert new File("$outputDir/salmon/WT_REP1/quant.sf").exists() },
                { assert new File("$outputDir/salmon/WT_REP1/quant.genes.sf").exists() },
                { assert new File("$outputDir/salmon/WT_REP2/quant.sf").exists() },
                { assert new File("$outputDir/salmon/WT_REP2/quant.genes.sf").exists() },
                { assert snapshot(path("$outputDir/star_rsem/bigwig/RAP1_IAA_30M_REP1.forward.bigWig"),
                                path("$outputDir/star_rsem/bigwig/RAP1_IAA_30M_REP1.reverse.bigWig"),
                                path("$outputDir/star_rsem/bigwig/RAP1_UNINDUCED_REP1.forward.bigWig"),
                                path("$outputDir/star_rsem/bigwig/RAP1_UNINDUCED_REP1.reverse.bigWig"),
                                path("$outputDir/star_rsem/bigwig/RAP1_UNINDUCED_REP2.forward.bigWig"),
                                path("$outputDir/star_rsem/bigwig/RAP1_UNINDUCED_REP2.reverse.bigWig"),
                                path("$outputDir/star_rsem/bigwig/WT_REP1.forward.bigWig"),
                                path("$outputDir/star_rsem/bigwig/WT_REP1.reverse.bigWig")).match("star_rsem/bigwig/") },
                { assert new File("$outputDir/star_rsem/bigwig/WT_REP2.forward.bigWig").exists() },
                { assert new File("$outputDir/star_rsem/bigwig/WT_REP2.reverse.bigWig").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.markdup.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.markdup.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.markdup.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_IAA_30M_REP1.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.markdup.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.markdup.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.markdup.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP1.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.markdup.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.markdup.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.markdup.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/RAP1_UNINDUCED_REP2.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.markdup.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.markdup.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.markdup.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP1.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.markdup.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.markdup.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.markdup.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.sorted.bam.flagstat").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.sorted.bam.idxstats").exists() },
                { assert new File("$outputDir/star_rsem/samtools_stats/WT_REP2.sorted.bam.stats").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_IAA_30M_REP1.gene.abundance.txt").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP1.gene.abundance.txt").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP2.gene.abundance.txt").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP1.gene.abundance.txt").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP2.gene.abundance.txt").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_IAA_30M_REP1.coverage.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP1.coverage.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP2.coverage.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP1.coverage.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP2.coverage.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_IAA_30M_REP1.transcripts.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP1.transcripts.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/RAP1_UNINDUCED_REP2.transcripts.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP1.transcripts.gtf").exists() },
                { assert new File("$outputDir/star_rsem/stringtie/WT_REP2.transcripts.gtf").exists() },
                { assert snapshot(path("$outputDir/star_rsem/RAP1_IAA_30M_REP1.genes.results"),
                                path("$outputDir/star_rsem/RAP1_IAA_30M_REP1.isoforms.results"),
                                path("$outputDir/star_rsem/RAP1_UNINDUCED_REP1.genes.results"),
                                path("$outputDir/star_rsem/RAP1_UNINDUCED_REP1.isoforms.results"),
                                path("$outputDir/star_rsem/RAP1_UNINDUCED_REP2.genes.results"),
                                path("$outputDir/star_rsem/RAP1_UNINDUCED_REP2.isoforms.results"),
                                path("$outputDir/star_rsem/WT_REP1.genes.results"),
                                path("$outputDir/star_rsem/WT_REP1.isoforms.results"),
                                path("$outputDir/star_rsem/WT_REP2.genes.results"),
                                path("$outputDir/star_rsem/WT_REP2.isoforms.results"),
                                path("$outputDir/star_rsem/deseq2_qc/R_sessionInfo.log")).match("star_rsem") },
                { assert snapshot(path("$outputDir/star_rsem/dupradar/gene_data/").list()).match("star_rsem/dupradar/gene_data/") },
                { assert snapshot(path("$outputDir/star_rsem/dupradar/intercepts_slope/").list()).match("star_rsem/dupradar/intercepts_slope/") },
                { assert new File("$outputDir/star_rsem/RAP1_IAA_30M_REP1.markdup.sorted.bam").exists() },
                { assert new File("$outputDir/star_rsem/RAP1_IAA_30M_REP1.markdup.sorted.bam.bai").exists() },
                { assert new File("$outputDir/star_rsem/WT_REP1.markdup.sorted.bam").exists() },
                { assert new File("$outputDir/star_rsem/WT_REP1.markdup.sorted.bam.bai").exists() },
                { assert new File("$outputDir/star_rsem/RAP1_UNINDUCED_REP1.markdup.sorted.bam").exists() },
                { assert new File("$outputDir/star_rsem/RAP1_UNINDUCED_REP1.markdup.sorted.bam.bai").exists() },
                { assert new File("$outputDir/star_rsem/WT_REP2.markdup.sorted.bam").exists() },
                { assert new File("$outputDir/star_rsem/WT_REP2.markdup.sorted.bam.bai").exists() },
                { assert new File("$outputDir/star_rsem/RAP1_UNINDUCED_REP2.markdup.sorted.bam").exists() },
                { assert new File("$outputDir/star_rsem/RAP1_UNINDUCED_REP2.markdup.sorted.bam.bai").exists() },
                { assert new File("$outputDir/star_rsem/log/RAP1_IAA_30M_REP1.log").exists() },
                { assert new File("$outputDir/star_rsem/log/RAP1_UNINDUCED_REP1.log").exists() },
                { assert new File("$outputDir/star_rsem/log/RAP1_UNINDUCED_REP2.log").exists() },
                { assert new File("$outputDir/star_rsem/log/WT_REP1.log").exists() },
                { assert new File("$outputDir/star_rsem/log/WT_REP2.log").exists() },
                { assert new File("$outputDir/star_rsem/picard_metrics/RAP1_IAA_30M_REP1.markdup.sorted.MarkDuplicates.metrics.txt").exists() },
                { assert new File("$outputDir/star_rsem/picard_metrics/RAP1_UNINDUCED_REP1.markdup.sorted.MarkDuplicates.metrics.txt").exists() },
                { assert new File("$outputDir/star_rsem/picard_metrics/RAP1_UNINDUCED_REP2.markdup.sorted.MarkDuplicates.metrics.txt").exists() },
                { assert new File("$outputDir/star_rsem/picard_metrics/WT_REP1.markdup.sorted.MarkDuplicates.metrics.txt").exists() },
                { assert new File("$outputDir/star_rsem/picard_metrics/WT_REP2.markdup.sorted.MarkDuplicates.metrics.txt").exists() },
                { assert new File("$outputDir/star_rsem/qualimap/RAP1_IAA_30M_REP1/rnaseq_qc_results.txt").exists() },
                { assert new File("$outputDir/star_rsem/qualimap/RAP1_UNINDUCED_REP1/rnaseq_qc_results.txt").exists() },
                { assert new File("$outputDir/star_rsem/qualimap/RAP1_UNINDUCED_REP2/rnaseq_qc_results.txt").exists() },
                { assert new File("$outputDir/star_rsem/qualimap/WT_REP1/rnaseq_qc_results.txt").exists() },
                { assert new File("$outputDir/star_rsem/qualimap/WT_REP2/rnaseq_qc_results.txt").exists() },
                { assert snapshot(path("$outputDir/star_rsem/rseqc/bam_stat/RAP1_IAA_30M_REP1.bam_stat.txt"),
                                path("$outputDir/star_rsem/rseqc/bam_stat/RAP1_UNINDUCED_REP1.bam_stat.txt"),
                                path("$outputDir/star_rsem/rseqc/bam_stat/RAP1_UNINDUCED_REP2.bam_stat.txt"),
                                path("$outputDir/star_rsem/rseqc/bam_stat/WT_REP1.bam_stat.txt"),
                                path("$outputDir/star_rsem/rseqc/infer_experiment/RAP1_IAA_30M_REP1.infer_experiment.txt"),
                                path("$outputDir/star_rsem/rseqc/infer_experiment/RAP1_UNINDUCED_REP1.infer_experiment.txt"),
                                path("$outputDir/star_rsem/rseqc/infer_experiment/RAP1_UNINDUCED_REP2.infer_experiment.txt"),
                                path("$outputDir/star_rsem/rseqc/infer_experiment/WT_REP1.infer_experiment.txt"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/bed/RAP1_IAA_30M_REP1.junction.bed"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/bed/RAP1_UNINDUCED_REP1.junction.bed"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/bed/RAP1_UNINDUCED_REP2.junction.bed"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/bed/WT_REP1.junction.bed"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/bed/WT_REP2.junction.bed"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/log/RAP1_IAA_30M_REP1.junction_annotation.log"),
                                path("$outputDir/star_rsem/rseqc/junction_annotation/log/WT_REP1.junction_annotation.log"),
                                path("$outputDir/star_rsem/rseqc/read_distribution/RAP1_IAA_30M_REP1.read_distribution.txt"),
                                path("$outputDir/star_rsem/rseqc/read_distribution/RAP1_UNINDUCED_REP1.read_distribution.txt"),
                                path("$outputDir/star_rsem/rseqc/read_distribution/RAP1_UNINDUCED_REP2.read_distribution.txt"),
                                path("$outputDir/star_rsem/rseqc/read_distribution/WT_REP1.read_distribution.txt")).match("star_rsem/rseqc/") },
                { assert new File("$outputDir/star_rsem/rseqc/bam_stat/WT_REP2.bam_stat.txt").exists() },
                { assert new File("$outputDir/star_rsem/rseqc/infer_experiment/WT_REP2.infer_experiment.txt").exists() },
                { assert new File("$outputDir/star_rsem/rseqc/junction_annotation/log/WT_REP2.junction_annotation.log").exists() },
                { assert new File("$outputDir/star_rsem/rseqc/read_distribution/WT_REP2.read_distribution.txt").exists() },
                { assert new File("$outputDir/trimgalore/RAP1_IAA_30M_REP1_1.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/RAP1_IAA_30M_REP1_2.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/RAP1_UNINDUCED_REP1.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/RAP1_UNINDUCED_REP2.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/WT_REP1_1.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/WT_REP1_2.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/WT_REP2_1.fastq.gz_trimming_report.txt").exists() },
                { assert new File("$outputDir/trimgalore/WT_REP2_2.fastq.gz_trimming_report.txt").exists() }
            )
        }

    }

}
