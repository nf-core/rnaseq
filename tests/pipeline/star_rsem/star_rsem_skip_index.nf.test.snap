{
    "star_rsem/dupradar/gene_data/": {
        "content": [
            "RAP1_IAA_30M_REP1_dupMatrix.txt:md5,8da31020df3c5b25167345609c6d08c5",
            "RAP1_UNINDUCED_REP1_dupMatrix.txt:md5,e0e8fdbe29f05f8c722a97180cf95a58",
            "RAP1_UNINDUCED_REP2_dupMatrix.txt:md5,7d96d6ddf1d12d43837b105865aeaafa",
            "WT_REP1_dupMatrix.txt:md5,802dd0de10d9118943869239f8659c78",
            "WT_REP2_dupMatrix.txt:md5,e97a3c8d2e606d7d4b40cd33eb0b96c4"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "samplesheet": {
        "content": [
            "samplesheet.valid.csv:md5,3d40bf3d424290cec0dbe9ee95b7dd12"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "software_versions": {
        "content": [
            "software_versions.yml:md5,48c196a8961afe40e848ee407dbb98fe"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "salmon": {
        "content": [
            "R_sessionInfo.log:md5,fb0da0d7ad6994ed66a8e68348b19676",
            "salmon_tx2gene.tsv:md5,b367acfc4b7df7fb68fb1b1d2956e27e"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "star_rsem/rseqc/": {
        "content": [
            "RAP1_IAA_30M_REP1.bam_stat.txt:md5,bca6551fa4e1f59c46ffb8d02380260c",
            "RAP1_UNINDUCED_REP1.bam_stat.txt:md5,1ef7ff325fb28e485ad2c6b64b0bd126",
            "RAP1_UNINDUCED_REP2.bam_stat.txt:md5,b302c18e643aa10c293b04c0194b276c",
            "WT_REP1.bam_stat.txt:md5,84eeac9b473e6ec3e15bb787545f170f",
            "RAP1_IAA_30M_REP1.infer_experiment.txt:md5,1f31ddae1f98e779e30d846cde3834de",
            "RAP1_UNINDUCED_REP1.infer_experiment.txt:md5,a2a8e31dbd0ebd0c12f3f968ffd5391b",
            "RAP1_UNINDUCED_REP2.infer_experiment.txt:md5,9ec058d3a4162fd1b4c6175e2106eef4",
            "WT_REP1.infer_experiment.txt:md5,d55921c0084806cdf49b9bd8653c8e09",
            "RAP1_IAA_30M_REP1.junction.bed:md5,b9db9f3d0c776cb380ef34950640c51c",
            "RAP1_UNINDUCED_REP1.junction.bed:md5,55b551e4f0a667d8fa6c666510b5f3b2",
            "RAP1_UNINDUCED_REP2.junction.bed:md5,ca6ae7c39136891b49925e384d035ec4",
            "WT_REP1.junction.bed:md5,0a01e7b7b31601e5425fbf4a992461e7",
            "WT_REP2.junction.bed:md5,c255ca019fe2846e865b10e686895803",
            "RAP1_IAA_30M_REP1.junction_annotation.log:md5,c80f76e01c9a8d6f7386ed3c86ad0116",
            "WT_REP1.junction_annotation.log:md5,596922838fd999430397e0dd8ba763d5",
            "RAP1_IAA_30M_REP1.read_distribution.txt:md5,90de22c225fe5ecff10726adae623009",
            "RAP1_UNINDUCED_REP1.read_distribution.txt:md5,d1248d1d29aad1a3b5ca799380f411e4",
            "RAP1_UNINDUCED_REP2.read_distribution.txt:md5,24a0a1c381db13692b837e85bec2a148",
            "WT_REP1.read_distribution.txt:md5,8d086ee6973bc29e2c64cccbe9a318ad"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "bbsplit": {
        "content": [
            "RAP1_IAA_30M_REP1.stats.txt:md5,a87292c6f7166096166a9015c2e63dac",
            "RAP1_UNINDUCED_REP1.stats.txt:md5,781e49024150a8313779c574b58ad035",
            "RAP1_UNINDUCED_REP2.stats.txt:md5,e455d0163bf2d988224b0c0092605213",
            "WT_REP1.stats.txt:md5,6eab6936804c1157d5e1c61c0562d7de"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "star_rsem/bigwig/": {
        "content": [
            "RAP1_IAA_30M_REP1.forward.bigWig:md5,0808768567cf5470e2281b0da2383637",
            "RAP1_IAA_30M_REP1.reverse.bigWig:md5,1f5230da4621d4f3921af4731de95d36",
            "RAP1_UNINDUCED_REP1.forward.bigWig:md5,f893bb913f22db26d07475f09b516369",
            "RAP1_UNINDUCED_REP1.reverse.bigWig:md5,78411dd80f607b7a624fbc484fd0fee5",
            "RAP1_UNINDUCED_REP2.forward.bigWig:md5,a8bb05aa1e0029deb72e491c60e59e04",
            "RAP1_UNINDUCED_REP2.reverse.bigWig:md5,65e9aefc9928bf43d7a6f520443e6315",
            "WT_REP1.forward.bigWig:md5,40a41e03064aa1e2a2fc00400a6dd9f8",
            "WT_REP1.reverse.bigWig:md5,823282b3f27b0c26db805f497f47bf9c"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "star_rsem": {
        "content": [
            "RAP1_IAA_30M_REP1.genes.results:md5,4c623b4348bd09c4b35da8ebd600a6f4",
            "RAP1_IAA_30M_REP1.isoforms.results:md5,25fbda8df6f00d1da47a7b119f740792",
            "RAP1_UNINDUCED_REP1.genes.results:md5,e24094ff0fa6c28e009603782b8a9a39",
            "RAP1_UNINDUCED_REP1.isoforms.results:md5,47980c682ae7b493fb7f036e7851b112",
            "RAP1_UNINDUCED_REP2.genes.results:md5,b21366cbc0af6b01448078ec48b08dfd",
            "RAP1_UNINDUCED_REP2.isoforms.results:md5,a6dcf29555007c6bdcdc2a6e8547afbc",
            "WT_REP1.genes.results:md5,56b34ff9873e5a400f7a062355f0825b",
            "WT_REP1.isoforms.results:md5,61288951a174e76832af8bad8537e32e",
            "WT_REP2.genes.results:md5,33cad1f3fc4691df77305176f18f5e89",
            "WT_REP2.isoforms.results:md5,b14ced722405787f6235b3aebd2587a4",
            "R_sessionInfo.log:md5,fb0da0d7ad6994ed66a8e68348b19676"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "star_rsem/dupradar/intercepts_slope/": {
        "content": [
            "RAP1_IAA_30M_REP1_intercept_slope.txt:md5,d280fe126a5e82d24121d8662fd5a161",
            "RAP1_UNINDUCED_REP1_intercept_slope.txt:md5,264a7d98307145830e0ff05701fbaad8",
            "RAP1_UNINDUCED_REP2_intercept_slope.txt:md5,2342b7d12b896d0981ae8001e1f1dc61",
            "WT_REP1_intercept_slope.txt:md5,9c2e2b7890427e79c97e65827102d965",
            "WT_REP2_intercept_slope.txt:md5,db7a36459f2036de6657e74d2c98a47c"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    },
    "multiqc": {
        "content": [
            "multiqc_cutadapt.txt:md5,aac9581a5670cb55edf564f3d6c1f9a7",
            "multiqc_fastqc.txt:md5,68bdc9b37911a79484e0333e82679fa3",
            "picard_histogram.txt:md5,c9aa8a5ac6841ffb3a6cc2de45b44797"
        ],
        "timestamp": "2023-05-08T18:52:47+0000"
    }
}